src/main/java/net/minecraftforge/event/anvil/AnvilUpdateEvent.java

```diff
@@ -17,7 +17,7 @@
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
  */
 
-package net.minecraftforge.event;
+package net.minecraftforge.event.anvil;
```

Comment:
Is this really necessary?

Reply:
I can move the Anvil events back out to the main event folder again if so pleases.
I decided to create a anvil package seeing as there are now 3 Anvil related events.

Reply:
Moving an existing event will break things for anyone using the old event. While it may be more organized, it makes an unneeded breaking change. You could just mark it with a `TODO: move to net.minecraftforge.event.anvil` and get it moved next major update.

Reply:
Not sure if this should be moved or not, but given there has not been a RB yet so breaking changes are still allowed, this ***is*** the next major update.

Reply:
Will move everything back and add todos then.
Currently re-evaluating some stuff either way.